packages/react-make-styles/src/index.ts

```diff
@@ -1,4 +1,4 @@
 export { ax, createDOMRenderer } from '@fluentui/make-styles';
 
-export { makeStyles } from './makeStyles';
+export { makeStylesCompat } from './makeStyles';
```

Comment:
```suggestion
export { makeStylesCompat } from './makeStylesCompat';
```

Let's rename a file, too. We will need it on next steps to introduce a new version of `makeStyles`